File: deaf_grandma.js
function deafGrandma() {

    let goodbyes = 0
    
    let message = "HEY KID!"
    while (true) {
        
        let talk = prompt(message)
        
        
        if (talk=== ""){
            message = "WHAT?!";
        } else if (talk === "GOODBYE!") {
            if (goodbyes < 1) {
                message ="LEAVING SO SOON?"
                goodbyes++
            } else if (goodbye < 2) {
                message = "LATER, SKATER!"
                break;
            }
            
        } else if (talk === talk.toUpperCase()) {
            message = "NO, NOT SINCE 1946!"
        } else {
            message = "SPEAK UP, KID!"
        }
    }
    
    
    
    
    
    
    

}

deafGrandma();

File: deaf_grandma.py
def deaf_grandma():
    counter = 0
    message = "HEY KID!"

    while True:
        talk = input(message + "\n")
        if talk == "":
            message = "WHAT?!"
        elif talk == "GOODBYE!":
            if counter == 0:
                message = "LEAVING SO SOON?"
                counter +=1
            elif counter == 1:
                message = "LATER, SKATER!"
                break
        elif talk == talk.upper():
            message = "NO, NOT SINCE 1946!"
        else: 
            message = "SPEAK UP, KID!"
            



deaf_grandma()
